painters/train_cnn.py

```diff
@@ -24,92 +27,95 @@
 PENULTIMATE_LAYER = 51
 PENULTIMATE_SIZE = 2048
 SOFTMAX_LAYER = 55
-SOFTMAX_SIZE = 1584
+SOFTMAX_SIZE = 1057
```

Comment:
Should we hardcode this change? Will it stay the same as we're moving back to a full dataset?